c2corg_ui/templates/helpers/view.html

```diff
@@ -111,7 +111,8 @@
       title = ''
       if 'title_prefix' in locale and locale['title_prefix']:
           title = locale['title_prefix'] + ' : '
-      title += locale['title']
+      if 'title' in locale and locale['title']:
+          title += locale['title']
```

Comment:
The problem is that the route does not have any activities in that version. Without activity, no fields (as defined in the [common project](https://github.com/c2corg/v6_common/blob/master/c2corg_common/fields_route.py#L50)) are returned. Activities are now a required field, but apparently there are old versions without. I will work on a fix.